add-capi (#42)
giantswarm/giantswarm-aws-account-prerequisites: capa-controller-role/README.md

> # Cluster API role 

Reviewer: "Cluster API role" or "Cluster API AWS provider role"?

> If you dont know what `INSTALLATION_NAME` value is suppose to be, ask Giant Swarm staff and they will provide it.

Reviewer: ```suggestion
If you don't know what the `INSTALLATION_NAME` value is supposed to be, ask Giant Swarm staff and they will provide it.
```

> # Create role and policy
> 

Reviewer: I'd remove this line and just add a simple explanation on what this is meant to do/create right after the first heading.

```suggestion
```

## with aws cli

> ### requirement

Reviewer: ```suggestion
### requirements
```
